mlctl/clis/common/utils.py

```diff
@@ -0,0 +1,214 @@
+import yaml
+
+from mlctl.plugins.awssagemaker.deploy import AwsSagemakerDeploy
+from mlctl.plugins.awssagemaker.train import AwsSagemakerTrain
+from mlctl.plugins.awssagemaker.process import AwsSagemakerProcess
+import mlctl.plugins.mlflow.metadata as MlflowPlugin
+from mlctl.plugins.azureml.train import AzureMlTrain
+from mlctl.plugins.azureml.deploy import AzureMlDeploy
+from mlctl.jobs.MlctlTrainJob import MlctlTrainJob
+from mlctl.jobs.MlctlDeployJob import MlctlDeployJob
+from mlctl.jobs.MlctlProcessJob import MlctlProcessJob
+
+def docker_instructions(image_name):
+    return f'\nThe container was built with tag {image_name}. \
+To start the container on a service, tag your image to your remote repository \
+and then push the container. \
+The Docker commands should look like:\n\n\
+docker tag {image_name} [remote_repository]\n\
+docker push [remote_repository]'
+
+def parse_process_yamls(job_config, provider_config=None):
```

Comment:
this is not scalable. We should have dedicated class for json schema and we can also use [jsonschema](https://python-jsonschema.readthedocs.io/en/stable/) to validate instead of the helper.py you have implemented

Reply:
I think this is what AzureML uses for their schema files. I think the framework we should think about this in is:

1)  Is the injection method of creating config file versus something like passing a python class the better approach
2) is JSON/YAML the right choice for template files? Alt. protobuf